minidump-synth/src/lib.rs

```diff
@@ -758,7 +782,8 @@ impl<T: ListItem> ExList<T> {
         let section = Section::with_endian(endian)
             .D32(12)
```

Comment:
```suggestion
            .D32(12 + padding as u32)
```

Reply:
I went with passing the entire header size and then figuring out the padding from there, it's harder to get it wrong then

Reply:
Thanks a lot for spotting what was wrong with this code